cpp/src/cider/benchmarks/HashMapBenchmark.cpp

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright(c) 2022-2023 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <unistd.h>
+#include <iomanip>
+#include <iostream>
+#include <random>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+#include <common/hashtable/FixedHashMap.h>
+#include <common/hashtable/HashMap.h>
+#include <common/hashtable/HashTable.h>
+#include <common/hashtable/HashTableAllocator.h>
+#include <tests/utils/Utils.h>
+
+using namespace cider::hashtable;
+
+enum GeneratePattern { Sequence, Random };
+
+#define DBMS_HASH_MAP_COUNT_COLLISIONS

```

Comment:
why need this?

Reply:
It's old code to enable collision counter if it is defined. Removed.